delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/projects/ProjectsDeletionStream.scala

```diff
@@ -39,8 +40,9 @@ class ProjectsDeletionStream(
     keyValueStoreConfig: KeyValueStoreConfig
 ) {
 
-  private val projectionId: CacheProjectionId = CacheProjectionId("ProjectsDeletionProgress")
-  private val logger: Logger                  = Logger[ProjectsDeletionStream]
+  private val projectionId: CacheProjectionId       = CacheProjectionId("ProjectsDeletionProgress")
+  implicit val tracingConfig: ProgressTracingConfig = ProgressTracingConfig(projectionId.value, Map.empty)
```

Comment:
the stream doesn't seem to make use of tracing

Reply:
It's best to make `implicit vals` private so that the compiler will report if it's not used.